### PR TITLE
Adding Stripe docs

### PR DESCRIPTION
**Pull request by AmanGuptAnalytics**
Added Stripe docs as updated by @AstrakhantsevaAA, please check @adrianbr

### CODE REVIEW

**Comment by AmanGuptAnalytics** [comment]
@adrianbr please approve this PR.

**Comment by AstrakhantsevaAA** [comment]
> @AmanGuptAnalytics @AstrakhantsevaAA please make sure that wording and file names conform with the change to verified sources

should we do this after the merge of [this](https://github.com/dlt-hub/verified-sources/pull/177) PR ?

**Comment by AmanGuptAnalytics** [comment]
@AstrakhantsevaAA, could you please review the latest commit? I have made adjustments to incorporate the updated terminology, replacing "pipelines" with "verified source" where applicable. However, due to certain constraints, it was not feasible to make this substitution in every instance. I made the change whenever it seemed appropriate. Kindly review the modifications. 

Please feel free to modify further so that we have a better idea.

File: docs/website/docs/pipelines/stripe.md [hunk 1]
```diff
@@ -42,20 +42,21 @@ To get started with your data pipeline, follow these steps:
 dlt init stripe_analytics bigquery
 ```
 
-This command will initialize your pipeline with Stripe as the source and BigQuery as the destination. If you'd like to use a different destination, simply replace **`bigquery`** with the name of your preferred destination. You can find supported destinations and their configuration options in our [documentation](https://dlthub.com/docs/destinations/duckdb)
+This command will initialize your verified source with Stripe as the source and BigQuery as the destination. If you'd like to use a different destination, simply replace **`bigquery`** with the name of your preferred destination. You can find supported destinations and their configuration options in our [documentation](https://dlthub.com/docs/destinations/duckdb)
```
Comment (AstrakhantsevaAA) [inline docs/website/docs/pipelines/stripe.md:45]: IMO: This command will initialize your verified source with Stripe and creates pipeline example with BigQuery as the destination.

File: docs/website/docs/pipelines/stripe.md [hunk 2]
```diff
@@ -115,7 +116,7 @@ INCREMENTAL_ENDPOINTS = ("Event", "Invoice", "BalanceTransaction")
 
 ### **Source and resource methods**
 
-`dlt` works on the principle of [sources](https://dlthub.com/docs/general-usage/source) and [resources](https://dlthub.com/docs/general-usage/resource) that for this pipeline are found in the `__init__.py` file within the *stripe_analytics* directory. This pipeline `stripe_analytics_pipeline.py` has three default methods that form the basis of loading the pipeline. The methods are :
+`dlt` works on the principle of [sources](https://dlthub.com/docs/general-usage/source) and [resources](https://dlthub.com/docs/general-usage/resource) that for this verfied source are found in the `__init__.py` file within the *stripe_analytics* directory. This verfied source `stripe_analytics_pipeline.py` has three default methods that form the basis of loading the pipeline. The methods are :
```
[the remark this answers is not in the document]
Comment (AstrakhantsevaAA) [reply docs/website/docs/pipelines/stripe.md]: > has three default methods that form the basis of loading the pipeline. The methods are :

has three examples how to use verified source. The examples are :

**Pull request closed without merging**